### ISSUE TITLE
Can't run sst start when handler filename has a dot in it

### ISSUE THREAD

**Issue by AlexBrandes** [serverless-stack/serverless-stack#600]
When using a handler file with a dot in the filename, `sst start` fails to find the handler. 

```
handler: 'src/users/index.controller.main'
```

With file:
```
src/users/index.controller.ts
```

Error:
```
Could not resolve "/project/src/users/index.js"
```

This should be a one liner to fix. I'm happy to submit a PR to address it if that would be helpful. 

Related to https://github.com/serverless-stack/serverless-stack/issues/583 but the PR open to fix it would not solve this problem.

**Comment by fwang** [comment]
Hey @AlexBrandes, sorry for the delay. I will take care of it today and cut a release with it!

**Comment by fwang** [comment]
Fixed in [v0.35.2](https://github.com/serverless-stack/serverless-stack/releases/tag/v0.35.2)
